src/main/scala/cromwell/util/TryUtil.scala

```diff
@@ -5,6 +5,11 @@ import java.io.{PrintWriter, StringWriter}
 import scala.util.{Failure, Try}
 
 object TryUtil {
+  implicit class PimpedOption[T](option: Option[T]) {
```

Comment:
I forgot I had put this in as I ended up not using it. If ppl think this is useful I'll leave it in otherwise I'll get rid of it